tests/unit/test_cassette_patterns.py

```
@@ -0,0 +1,434 @@
+def test_is_clean_flags_sid_starting_with_S_in_storage_state() -> None:
+    """Same I7 hole in Playwright ``storage_state.json`` shape."""
+    text = '{"name":"SID","value":"S_REAL_VALUE","domain":".google.com"}'
+    ok, leaks = is_clean(text)
+    assert not ok
+    assert any("SID" in leak for leak in leaks)
+
+
+def test_is_clean_flags_sid_starting_with_S_in_json_key() -> None:
+    """Same I7 hole in the JSON-dict-with-cookie-name-as-key shape."""
+    text = '{"SAPISID": "S_real_leaked_token_here"}'
+    ok, leaks = is_clean(text)
```

Ruff (0.15.12)

[warning] Unpacked variable `leaks` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)